WORKSPACE

```diff
@@ -132,15 +163,21 @@ workspace_refs(
 # Load NPM dependencies #
 #########################
 
+# Load //builder/nodejs
 load("@vaticle_dependencies//builder/nodejs:deps.bzl", nodejs_deps = "deps")
-nodejs_deps(["@vaticle_dependencies//builder/nodejs:remove-node-patches.patch"])
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
+nodejs_deps()
+
+load("@build_bazel_rules_nodejs//:repositories.bzl", "build_bazel_rules_nodejs_dependencies")
```

Comment:
we are using the last version of `build_bazel_rules_nodejs`, in the future we should migrate to `rules_nodejs`